manopt/solvers/arc/arc_lanczos.m

```diff
@@ -90,7 +90,7 @@
         eta = M.zerovec(x);
         Heta = eta;
         stop_str = 'Cost gradient is zero';
-        stats = struct('newton_iterations', 0);
+        stats = struct('newton_iterations', 0, 'gradnorms', 0, 'func_values', 0);
```

Comment:
It's great that these are logged here too -- can you make sure that this is documented in this file, same as for CG and GD?